data/data/ovirt/masters/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "ovirt_vm" "master" {
+  count       = var.master_count
+  name        = "master-${count.index}.${var.cluster_domain}"
+  cluster_id  = var.ovirt_cluster_id
+  template_id = var.ovirt_template_id
+  memory      = "12288"
```

Comment:
We should probably track somewhere that this memory and cores should be configurable using the
a) the master Machine objects
b) _maybe_ using the machine pool in install-config.yaml